src/components/home/Home.js

```diff
@@ -104,7 +138,35 @@ function Home() {
         <Typography variant="h3" align="center">
           {t('Browse Open Educational Resources')}
         </Typography>
-        <Search handleSearch={handleSearch} isLoading={isLoading} />
+        <Search
+          handleSearch={handleKeywordInput}
+          handleClick={handleClick}
+          isLoading={isLoading}
+        />
+        <FormControl component="fieldset" className={classes.searchOptions}>
+          <FormLabel component="legend">Search Range</FormLabel>
+          <RadioGroup
+            row
+            aria-label="range"
+            name="search-range-radio-button-group"
+            value={range}
+            onChange={handleRangeChange}
+          >
+            <FormControlLabel value="all" control={<Radio />} label="All" />
+            <FormControlLabel value="title" control={<Radio />} label="Title" />
+            <FormControlLabel value="tag" control={<Radio />} label="Tag" />
+            <FormControlLabel
+              value="author"
+              control={<Radio />}
+              label="Author"
+            />
+            <Tooltip title="Use | or & for union/intersection of multiple keywords if choose 'Tag' or 'All'">
```

Comment:
Not sure we should let the user add such syntax. I think we need to search for some examples, I don't know if a button would be intuitive. We can discuss with Paul (designer) as well. For now I would assume an intersection.

Reply:
Ok I would assume intersection, and I'll create a new issue for this feature